sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -1676,9 +1722,10 @@ def hpat_pandas_series_corr(self, other, method='pearson', min_periods=None):
     Returns
     -------
     :obj:`float`
-         returns :obj:`float` object
+        returns :obj:`float` object
     """
 
+    _func_name = 'Method corr().'
```

Comment:
Why did you add the line? It seems unnecessary.

Reply:
This is necessary so that all functions are in the same style.

Reply:
Now I see it's used and necessary.